core/runtime.rs

```diff
@@ -4871,4 +4871,39 @@ Deno.core.opAsync("op_async_serialize_object_with_numbers_as_keys", {
       ..Default::default()
     });
   }
+
+  #[test]
+  fn ops_in_js_have_proper_names() {
```

Comment:
**praise:** Nice to have test for this, even though it's a relatively minor thing.

Reply:
Agreed!